### PR TITLE
Add another function that solves the second part of the puzzle.

### PR DESCRIPTION
**Pull request by exo-cortex**
Now the program only solves part 2 of the puzzle. 
But that is OK.

### CODE REVIEW

File: day_01/src/main.rs [hunk 1]
```diff
@@ -24,12 +24,46 @@ fn task_1(reader: &mut BufReader<File>) -> u32 {
...
+    }
+    sum
+}
+
 fn main() {
     let mut example_input = BufReader::new(File::open("./example_1.txt").unwrap());
     let mut full_input = BufReader::new(File::open("./puzzle_input.txt").unwrap());
 
-    println!("example data: total sum is {}", task_1(&mut example_input));
+    // println!("example data: total sum is {}", task_1(&mut example_input));
```
Comment (grandchild) [inline day_01/src/main.rs]: Please remove the commented code. It's in the history anyway if you need it.

File: day_01/src/main.rs [hunk 2]
```diff
@@ -24,12 +24,46 @@ fn task_1(reader: &mut BufReader<File>) -> u32 {
...
+            .filter(|c| c.is_digit(10))
+            .collect::<Vec<char>>();
+        // println!(" {:?}", &found_numbers);
+        let combined_number = format!(
+            "{}{}",
+            found_numbers.first().unwrap().to_string(),
+            found_numbers.last().unwrap().to_string()
+        )
+        .parse::<u32>()
+        .unwrap();
```
Comment (grandchild) [inline day_01/src/main.rs]: Use `expect` maybe?
```suggestion
        .expect("Should be a u32 number");
```

**Review by grandchild — CHANGES_REQUESTED** [review]
I found some issues, please have a look.

File: day_01/src/main.rs (hunk 1 above)
Comment (exo-cortex) [reply day_01/src/main.rs]: as always - you're right ...

File: day_01/src/main.rs (hunk 2 above)
Comment (exo-cortex) [reply day_01/src/main.rs]: good idea!

**Review by grandchild — APPROVED** [review]
lgtm

**Pull request merged**